main.go

```diff
@@ -470,7 +474,7 @@ func readSettingsFile() bool {
 			log.Println("Client secret need to be a 52 char alphanumeric string")
 			return false
 		}
-		if settings.MaxCacheSizeInMebibytes < 10240 {
+		if settings.MaxCacheSizeInMebibytes < 10240*1024*1024 {
```

Comment:
is this not 10485760 gibibits?

Reply:
Well Kind of got lazy there, and I think the compiler might optimise this constant,

If it does not you only do that at the launch of the program so...

Reply:
oh duhh, yeah I forgot about that, thanks.